src/app/main.c

```diff
@@ -242,6 +252,36 @@ main (int    argc,
 
   if (!command->fn)
     {
+      command = preview_commands;
+      while (command->name)
+        {
+          if (g_strcmp0 (command_name, command->name) == 0)
+            {
+              g_printerr ("notice: %s is a preview command and subject to change.\n",
```

Comment:
Super minor nit: could the notices be capitalized? Regular output from rpm-ostree and ostree are capitalized.

I'd also print in bold here or something. It's easy to miss it in commands that output a lot. But I guess that's more subjective (since people will be running it for the first time presumably, they'd probably read every line of the output).

/bikeshed